tests/integration/test_full_pipeline.py

```
@@ -0,0 +1,410 @@
+        orchestrator = ReviewOrchestrator()
+        
+        # Track checkpoint calls
+        checkpoint_count = 0
+        
+        def mock_input(prompt):
+            nonlocal checkpoint_count
+            checkpoint_count += 1
+            return 'y'  # Always continue
+        
+        with patch('builtins.input', side_effect=mock_input):
+            result = orchestrator.review_contract(sample_contract_path)
```

Ruff (0.12.2)

Local variable `result` is assigned to but never used

Remove assignment to unused variable `result`

(F841)